integration-tests/contract-empty-implicit-account-into-new-implicit-account.spec.ts

```diff
@@ -38,66 +37,24 @@ CONFIGS().forEach(({ lib, rpc, setup, createAddress, protocol }) => {
             // // Emptying the account
             const totalFees = estimate.suggestedFeeMutez + estimate.burnFeeMutez;
             const maxAmount = balance.minus(totalFees).toNumber();
-            // // Temporary fix, see https://gitlab.com/tezos/tezos/-/issues/1754
-            // // we need to increase the gasLimit and fee returned by the estimation
+
+            // Temporary fix, see https://gitlab.com/tezos/tezos/-/issues/1754
+            // we need to increase the gasLimit and fee returned by the estimation
             const gasBuffer = 500;
             const MINIMAL_FEE_PER_GAS_MUTEZ = 0.1;
             const increasedFee = (gasBuffer: number, opSize: number) => {
                  return (gasBuffer) * MINIMAL_FEE_PER_GAS_MUTEZ + opSize
              }
 
             const opTransfer = await sender.contract.transfer({
-                 to: receiver_pkh,
-                 mutez: true,
-                 amount: maxAmount - increasedFee(gasBuffer, Number(estimate.opSize)),
-                 fee: estimate.suggestedFeeMutez + increasedFee(gasBuffer, Number(estimate.opSize)), // baker fees
-                 gasLimit: estimate.gasLimit + gasBuffer,
-                 storageLimit: estimate.storageLimit 
-             });
-
-             await opTransfer.confirmation();
-             const finalBalance = await Tezos.tz.getBalance(sender_pkh);
-
-             expect(finalBalance.toString()).toEqual("0")
-
-            done();
-        });
-
-        kathmandunet('on K protocol, reveals the sender account, creates an unrevealed implicit account, empties the sender account into the created one', async (done) => {
```

Comment:
The distinction between J and K protocols should be kept.

Reply:
restored the two tests J and K